tla-io/src/main/scala/at/forsyte/apalache/io/quint/Quint.scala

```diff
@@ -331,6 +331,18 @@ class Quint(moduleData: QuintOutput) {
         case "item"   => binaryApp(opName, tla.app)
         case "tuples" => variadicApp(tla.times)
 
+        // Maps (functions)
+        // case "Map"       => null // Rewire with "Apalache!SetAsFun"
+        // case "Map"       => variadicApp(args => tla.appOp("Apalache!SetAsFun", tla.enumSet(args: _*)))
+        case "get"       => binaryApp(opName, tla.app)
+        case "keys"      => unaryApp(opName, tla.dom)
+        case "mapBy"     => null
+        case "setToMap"  => unaryApp(opName, tla.setAsFun)
+        case "setOfMaps" => binaryApp(opName, tla.funSet)
+        case "set"       => ternaryApp(opName, tla.except)
+        case "setBy"     => null
+        case "put"       => null
```

Comment:
This requires special code, as we discussed with @thpani

Reply:
I will address `put` separately in #2493 /cc @shonfeder